dynamiqs/solver.py

```diff
@@ -34,12 +34,14 @@ def __init__(
         safety_factor: float = 0.9,
         min_factor: float = 0.2,
         max_factor: float = 5.0,
+        max_steps: int = 100_000,
```

Comment:
The lower we put as default the faster it will be. Cf https://github.com/patrick-kidger/diffrax/issues/161 (a must read!). cc @gautierronan

Reply:
Good to know !